apidoc/Titanium/UI/WebView.yml

```diff
@@ -1114,6 +1156,67 @@ properties:
     description: May be undefined.
     type: String
 
+---
+name: DataCreationResult
```

Comment:
Extend `ErrorResponse`? (I guess we don't set a `code` property, though?)

May make more sense to align this consistently, though, by doing so.